data/mods/My_Sweet_Cataclysm/sweet_items.json

```diff
@@ -68,5 +68,21 @@
     "description": "Sweet, sweet sugar.  Bad for your teeth and surprisingly not very tasty on its own.",
     "material": [ "junk" ],
     "vitamins": [ [ "junk_allergen", 1 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "comestible_type": "FOOD",
+    "id": "bone_sugar",
+    "name": { "str_sp": "sugar bone" },
+    "description": "A large bone, made entirely up of super-compressed sugar.  Way too hard to be eaten directly, but it can be broken down into sugar.",
```

Comment:
```suggestion
    "description": "A large bone, made entirely of super-compressed sugar.  Way too hard to be eaten directly, but it can be broken down into sugar.",
```